lib/node_modules/@stdlib/utils/is-lowercase/lib/is_lowercase.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// MODULES //
+
+var isString = require( '@stdlib/utils/is-string' ).isPrimitive;
+
+
+// MAIN //
+
+/**
+* Tests if a value is a lowercase string.
+*
+* @param {*} value - value to test
+* @returns {boolean} boolean indicating whether value is a lowercase string
+*
+* @example
+* var bool = isLowercase( 'salt and light' );
+* // returns true
+* @example
+* var bool = isLowercase( 'HELLO' );
+* // returns false
+* @example
+* var bool = isLowercase( 'World' );
+* // returns false
+* @example
+* var bool = isLowercase( '!' );
+* // returns true
+*/
+function isLowercase( value ) {
+	return isString( value ) && ( value.valueOf() === value.toLowerCase() );
```

Comment:
If checking for primitives, why use `valueOf()`? Also can use `lowercase` string util.